janitor/src/main/scala/com/broadinstitute/dsp/janitor/DbReader.scala

```diff
@@ -141,6 +142,25 @@ object DbReader {
         """
       .query[BucketToRemove]
 
+  /**
+   * We want to flag apps that have been stuck in creating for more than 2 hours and report them to the user
+   * to avoid hidden costs. Since apps cannot be deleted while creating, there is not much more we can do other
+   * than report.
+   * We sinilarily want to flag apps that have been stuck in deleting for more than 1 hour.
+   *
+   * Note that there are no `Creating` status for apps, only `STARTING`, `PRESTARTING`, `PRECREATING`, or `PROVISIONING`
```

Comment:
@Qi77Qi would you know if `starting` corresponds to any or all of these statuses?

Reply:
From @jdcanas it should only be `PRECREATING` and `PROVISIONING` 👍